### ISSUE TITLE
feat(tests): gradient tests, drawer tests, CPU/GPU tests

### ISSUE THREAD

**Issue by moesio-f** [moesio-f/py-benchmark-functions#10]
# Summary

Currently, Tensorflow and PyTorch implementations allow for  calculating the gradients of functions. However, this functionality is neither tested nor commented in the README/docs . Thus, it would important to test this functionality for the implemented functions and update the documentation accordingly. Additionally, the drawer utility should also be tested alongside with `torch` CPU/GPU testing.

**Comment by moesio-f** [comment]
Partially implemented in #11. Still missing drawer and gradient tests.
